benchmarks/bench_sampling_hy3.py

```
@@ -0,0 +1,104 @@
+        penalty = torch.zeros(
+            (batch_size, (vocab_size + 7) // 8), dtype=torch.uint8, device="cuda"
+        )
+        slots = torch.arange(batch_size, dtype=torch.int32, device="cuda")
+
+        temperature_call = lambda: flashinfer.sampling.fused_sampling_from_logits_hy3(
+            logits,
+            workspace_buffer=workspace,
+            out=output,
+            temperature=1.05,
+            gumbel_noise=gumbel,
+        )
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `temperature_call` as a `def`

(E731)